src/main/java/Aly.java

```diff
@@ -0,0 +1,242 @@
+import java.util.Scanner;
+
+public class Aly {
+
+    private static final int MAX_TASKS = 100;
+    private static final String LINE_SEPARATOR = "-----------------------------------------------------------------------------";
+    private static final String LOGO = "    _      _     _   _\n"
+            + "   / \\    | |   \\ \\ / /\n"
+            + "  / _ \\   | |    \\ V / \n"
+            + " / ___ \\  | |__   | |  \n"
+            + "/_/   \\_\\ |____|  |_|  \n";
+    private static Scanner in = new Scanner(System.in);
+    private static Task[] tasks = new Task[MAX_TASKS];
+
+    private static void printLine() {
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    private static void initialise() {
+        boolean isExit = false;
+        while (!isExit) {
+            printLine();
+            System.out.println("I'm hungry, what do you want?");
+            System.out.println("Faster lah! Enter 1 for echo function, 2 for task creation function, 3 for task marking function, 0 to exit");
```

Comment:
The indentation for wrapped lines should be 8 spaces more than the parent line. The split lines are not indented here.